scripts/circle-ci/run-circle-tests.sh

```diff
@@ -9,7 +9,7 @@ function checkDocsBuild {
   make html
 }
 
-CONTAINER_1=(':atlasdb-cassandra-integration-tests:check')
+CONTAINER_1=(':atlasdb-cassandra-integration-tests:check :atlasdb-jepsen-tests:check')
```

Comment:
Nope! This is just the unit tests for the classes I added. Which container should we run it on?